spec/jobs/facilities/pssg_download_spec.rb

```diff
@@ -53,15 +53,15 @@
     end
 
     it 'leaves facility with original drive time band' do
-      create :sixty_mins_648A4
+      existing_drive_time = create :sixty_mins_648A4
 
       allow(pssg_client_stub).to receive(:get_drivetime_bands).with(0, 1).and_return(drive_time_data_402)
       subject.perform
       expect(BaseFacility.find_facility_by_id('vha_648A4').drivetime_bands).not_to be_nil
       expect(BaseFacility.find_facility_by_id('vha_648A4').drivetime_bands.size).to be(1)
-      expect(BaseFacility.find_facility_by_id('vha_648A4').drivetime_bands[0].name).to eql('drive_time_band_^0')
+      expect(BaseFacility.find_facility_by_id('vha_648A4').drivetime_bands[0].name).to eql('648A4 : 0 - 10')
       expect(BaseFacility.find_facility_by_id('vha_648A4').drivetime_bands[0].unit).to eql('minutes')
-      expect(DrivetimeBand.find_by(vha_facility_id: '648A4').name).to eql('drive_time_band_^0')
+      expect(DrivetimeBand.find_by(vha_facility_id: '648A4').polygon.to_s).to eql(existing_drive_time.polygon.to_s)
```

Comment:
the `eql` matcher is apparently meant for integers. I suggest using `eq` instead

Reply:
You might be able to compare the polygon objects with the `eq` matcher instead of converting them to strings, too. Maybe. Not sure